pkgs/development/libraries/litehtml/default.nix

```diff
@@ -16,6 +16,12 @@ stdenv.mkDerivation rec {
     hash = "sha256-9571d3k8RkzEpMWPuIejZ7njLmYstSwFUaSqT3sk6uQ=";
   };
 
+  # Don't search for non-existant gumbo cmake config
+  postPatch = ''
+    substituteInPlace cmake/litehtmlConfig.cmake \
+      --replace "find_dependency(gumbo)" ""
+  '';
+
```

Comment:
Why is this here?

Reply:
Gumbo don't provide cmake config, `find_dependency(gumbo)` will cause failure

Archlinux does something similar： https://github.com/archlinux/svntogit-packages/blob/1cc60cd7bed3034608eb137ff82761e7ec88acb5/trunk/PKGBUILD#LL29C48-L29C56

Reply:
I mean why is this in a pr that doesn't touch anything in the litehtml closure

Also it isn't failing: https://hydra.nixos.org/build/213914327

Reply:
qtcreator use cmake to search litehtml, need this to ensure litehtml can be found. 
```
-- The following packages have not been found:

 * litehtml
```
Here should be a separate pr： https://github.com/NixOS/nixpkgs/pull/223361

Reply:
> Also it isn't failing: [hydra.nixos.org/build/213914327](https://hydra.nixos.org/build/213914327)

This does not affect the building of litehtml itself, but the software that depends on litehtml, we can also substitute in `postInstall`